assets/js/app.js

```diff
@@ -20,6 +20,7 @@ let header = $(`
    <li class="nav-item"><a class="nav-link" href="projects.html">Projects</a></li>
    <li class="nav-item"><a class="nav-link" href="research.html">Research</a></li>
    <li class="nav-item"><a class="nav-link" href="education.html">Education</a></li>
+   <li class="nav-item"><a class="nav-link" href="assets/Resume/jhon_doe.pdf" target="_blank">Resume</a></li>
```

Comment:
- Change the path to `./assets/docs/john_doe.pdf` and update the same in the actual asset itself. 
- Instead of a blank document. Upload the famous Deedy Resume template ( by changing the name to John Doe)